docs/src/pages/landing-page/Index.vue

```diff
@@ -196,4 +304,22 @@ export default defineComponent({
 .support {
   height: 100vh;
 }
+
+.my-grid {
+  font-size: .5rem;
```

Comment:
Any reason why you started using `rem`s here instead of pixels?

Reply:
So I'm thinking in situations where body's 1rem is 16px that should equal about 8px. And in situations where it is say less, that should scale down automatically. I will move it back to the px to adhere to the px style so far. And handle such responsiveness later on.